installing/disconnected_install/installing-mirroring-creating-registry.adoc

```diff
@@ -40,7 +40,9 @@ include::modules/mirror-registry-troubleshooting.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
```

Comment:
I know this is existing text and outside of the scope of the issue, this heading should be promoted, as the resources apply to the whole assembly. 

```suggestion
[id="additional-resources_installing-mirroring-creating-registry"]
== Additional resources```